### PR TITLE
Update aRMT specs 

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add COVID-19 questionnaires to aRMT specs (COVID-19 Diagnosis, COVID-19 Symptoms, COVID-19 CNS Baseline, COVID-19 CNS Followup)

### CODE REVIEW

File: specifications/active/aRMT-1.5.2.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/rpq/rpq_armt.json
+  - type: ART_COGNITIVE_TEST
+    topic: questionnaire_art_cognitive_test
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/art_cognitive_test/art_cognitive_test_armt.json
+  - type: COVID19_DIAGNOSIS
+    topic: questionnaire_covid19_diagnosis
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/covid19_diagnosis/covid19_symptoms_armt.json
```
Comment (yatharthranjan) [inline specifications/active/aRMT-1.5.2.yml:110]: the url should be for diagnosis

File: specifications/active/aRMT-1.5.2.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  - type: COVID19_SYMPTOMS
+    topic: questionnaire_covid19_symptoms
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/covid19_symptoms/covid19_symptoms_armt.json
+  - type: COVID19_RADARCNS_BASELINE
+    topic: questionnaire_covid19_radarcns_baseline
```
Comment (yatharthranjan) [inline specifications/active/aRMT-1.5.2.yml:116]: By following convention from other topic names like this, I think the study name should be before the questionnaire name, so `questionnaire_radarcns_covid19_baseline`

File: specifications/active/aRMT-1.5.2.yml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/covid19_radarcns_baseline/covid19_radarcns_baseline_armt.json
+  - type: COVID19_RADARCNS_FOLLOWUP
+    topic: questionnaire_covid19_radarcns_followup
```
Comment (yatharthranjan) [inline specifications/active/aRMT-1.5.2.yml:120]: Similar naming here too

File: specifications/active/aRMT-1.5.2.yml [hunk 4]
```diff
@@ -0,0 +1,134 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.5.2
```
Comment (yatharthranjan) [inline specifications/active/aRMT-1.5.2.yml:4]: Please update the minor version, so `1.6.0`.

File: specifications/active/aRMT-1.5.2.yml (hunk 2 above)
Comment (mpgxvii) [reply specifications/active/aRMT-1.5.2.yml:116]: ok should I change the questionnaire name as well for consistency?

Comment (mpgxvii) [reply specifications/active/aRMT-1.5.2.yml]: Made the changes. I've also changed the questionnaire name for consistency.

**Pull request merged**